package.json

```diff
@@ -12,6 +12,7 @@
     "base-64": "0.1.0",
     "commonmark": "github:mattermost/commonmark.js#c9c29834005af6eedd1a979d913b75e34f16a83b",
     "commonmark-react-renderer": "mattermost/commonmark-react-renderer#86fa63f898802953842526c2030f3b63c5d1ae7a",
+    "date-time-format-timezone": "^1.0.21-alpha.2",
```

Comment:
can we make this a fix version?

Reply:
Oh and please add this to the NOTICES.txt file

Reply:
yup. I'll get that fixed 👍